### ISSUE TITLE
Mouse editing of Price Marker

### ISSUE THREAD

**Issue by ghost** [rrag/react-stockcharts#338]
Hello rrag, Is there a callback for visual editing of price marker? So once the marker is set, you could drag the marker up or down to adjust the price manually? I was thinking about a workaround - onClick event on PriceCoordinate would trigger editing mode of the marker -> then getting new price level from charts onClick callback -> updating marker price. Drag callback would be cool though.

btw. thank you for your work, this library is really awesome!

**Comment by rrag** [comment]
This is not there right now, Recently I have been working on the interactive indicators and I am close to getting making the api final, These interactive indicators have click, doubleclick and drag handlers. Look at the [text](http://rrag.github.io/react-stockcharts/documentation.html#/interactive_text) example

I was hoping to create a new interactive indicator similar to text. If you are interested in taking a stab at creating a new interactive indicator for this it would be great.

**Comment by davidklodner** [comment]
Interactive text looks promising. I don't know your library inside out, but I'll do my best. From a quick look, it should be possible to combine interactive text and interactive components like Straight Line into what I was hoping for, am I right?

**Comment by jonolo6** [comment]
Anything I could do to help here? I'm interested in similar functionality. I'm decent with JS, but not a wiz.

**Comment by flavioespinoza** [comment]
I achieved this with Redux and a numeric input. 

I placed a price indicator with a price and subsequently set the Store price using Redux reducer and action methods.
<img width="985" alt="screen shot 2018-03-19 at 9 59 47 pm" src="https://user-images.githubusercontent.com/2214913/37635389-ebe56fe8-2bc0-11e8-899a-567c7a517ead.png">

I then set the initial price in my EditPrice Component using Redux's Provider & Connect pattern
<img width="861" alt="screen shot 2018-03-19 at 10 05 29 pm" src="https://user-images.githubusercontent.com/2214913/37635619-41d6b0b4-2bc2-11e8-9e20-f3d29cd57f3b.png">

I then created a function in my EditPrice Component to update the price in the Redux Store
<img width="1033" alt="screen shot 2018-03-19 at 9 51 22 pm" src="https://user-images.githubusercontent.com/2214913/37635436-23dea432-2bc1-11e8-8708-4a3c32b53824.png">

I then returned an input that updated the price during the onChange command.
<img width="644" alt="screen shot 2018-03-19 at 9 50 33 pm" src="https://user-images.githubusercontent.com/2214913/37635466-5926b1f2-2bc1-11e8-93e3-a5104caf5ecb.png">

Hope this helps :)



6d0ec551baa7.png">
